Breeze.UI/src/app/shared/classes/error.ts

```diff
@@ -3,8 +3,23 @@ import { DialogOptions } from '../classes/dialog-options';
 
 export class Error {
     static getMessage(error: any): string {
-        const firstError = Error.getFirstError(error);
-        return firstError.description || firstError.message;
+      if (!error) {
+        return null;
+      }
+
+      if (typeof error === 'string' || error instanceof String) {
+        return error.toString();
+      }
+
+      const firstError = Error.getFirstError(error);
+
+      // TODO: at the moment we have to check if detailed description contains word "Exception"
+      // as fullnode code abuses Description property for both exception stacktrace and detailed messages
+      // once this is fixed in fullnode this check can be removed. This has been discussed between
+      // Igor and Carlton on 2018-05-23
+      return !!firstError.description && firstError.description.indexOf('Exception:') < 0
```

Comment:
ES6's `!firstError.description.includes('Exception:')` would be more readable, maybe not _quite_ as performant

Reply:
Well this is built to ES5 so we can't use, nevermind